packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -2207,6 +2210,7 @@ export interface SendBoxStylesProps extends BaseCustomStyles {
 
 // @public
 export interface StatefulCallClient extends CallClient {
+    createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
```

Comment:
We don't need a new api here - it is declarative behavior so it should be provided by CallClient(SDK one), and you also have type DeclarativeCallAgent = CallAgent, so just leave the createCallAgent one, and it should be coming from CallClient directly

Reply:
Noticed things are slightly different on beta version... Feel free to close this one